types/audiosprite/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for audiosprite 0.6
+// Project: https://github.com/tonistiigi/audiosprite
+// Definitions by: Gyusun Yeom <https://github.com/Perlmint>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace audiosprite;
+export = audiosprite;
+
+declare function audiosprite(files: string[], option: audiosprite.Option, callback: (error: Error, obj: audiosprite.Result) => void): void;
```

Comment:
```js
  if (arguments.length == 2) {
    callback = arguments[1]
  } else if (arguments.length >= 3) {
    opts = arguments[1]
    callback = arguments[2]
  }
```

This function needs an overload without the `options` argument; it can't just be made optional since the function checks for `arguments.length`.